src/ConvertESOMTR/mainModule.f90

```diff
@@ -430,6 +430,7 @@ Subroutine readESOdictionary()
 !'check to make sure the entire file wasn't treated as the data dictionary
 If (lineOfESO /= 'End of Data Dictionary') Then
```

Comment:
@rraustad are you willing to "review" this massive change? If so I will point the Pivotal ticket to you.